cypress/fixtures/fireboltCalls/localization.js

```diff
@@ -60,3 +61,292 @@ exports.LOCALIZATION = {
+  method: 'manage_localization.addAdditionalInfo',
+  params: { key: null, value: 'value1' },
+  expected: 'error',
+};
+
+exports.SET_LOCALIZATION_ADDADDITIONALINFO_WITH_KEY_AND_VALUE_AS_NULL = {
+  method: 'manage_localization.addAdditionalInfo',
+  params: { key: null, value: null },
+  expected: 'error',
+};
+
+exports.SET_LOCALIZATION_ADDADDITIONALINFO_WITH_KEY_AND_VALUE_AS_INTEGER = {
+  method: 'manage_localization.addAdditionalInfo',
+  params: { key: 123, value: 123 },
+  expected: 'error',
+};
+
+exports.SET_LOCALIZATION_ADDADDITIONALINFO_WITH_KEY_AND_VALUE_AS_OBJECT = {
+  method: 'manage_localization.addAdditionalInfo',
+  params: { key: {}, value: {} },
+  expected: 'error',
+};
+
+exports.SET_LOCALIZATION_ADDADDITIONALINFO_WITH_EMPTY_OBJECT = {
+  method: 'manage_localization.addAdditionalInfo',
+  params: {},
+  expected: 'error',
+};
+
+exports.INVALID_PARAMETERS_FOR_LOCALIZATION_ADDADDITIONALINFO = {
+  method: 'localization.addAdditionalInfo',
+  validationJsonPath: 'result',
+  content: errorContent.INVALID_TYPE_PARAMS,
+  expectingError: true,
+};
+
+exports.GET_LOCALIZATION_ADDITIONALINFO = {
+  method: 'localization.additionalInfo',
+  params: {},
+};
+
+exports.EXPECTED_LOCALIZATION_ADDITIONALINFO = {
+  method: 'localization.additionalInfo',
+  validationJsonPath: 'result',
+  content: null,
+};
+
+exports.SET_LOCALIZATION_ADDADDITIONALINFO_WITH_STRING = {
+  method: 'manage_localization.addAdditionalInfo',
+  params: { key: 'defaultKey', value: 'defaultValue=' },
+};
+
+exports.SET_LOCALIZATION_ADDADDITIONALINFO_WITH_EMPTY_STRING = {
+  method: 'manage_localization.addAdditionalInfo',
+  params: { key: '', value: '' },
+};
+
+exports.SET_LOCALIZATION_REMOVEADDITIONALINFO_WITH_STRING = {
+  method: 'manage_localization.removeAdditionalInfo',
+  params: { key: 'defaultKey' },
+};
+
+exports.SET_LOCALIZATION_REMOVEADDITIONALINFO_WITH_EMPTY_STRING = {
+  method: 'manage_localization.removeAdditionalInfo',
+  params: { key: '' },
+};
+
+exports.STRING_FOR_LOCALIZATION_ADDITIONALINFO = {
+  method: 'localization.additionalInfo',
+  validationJsonPath: 'result',
+  content: { defaultKey: 'defaultValue=' },
+};
+
+exports.EMPTY_STRING_FOR_LOCALIZATION_ADDITIONALINFO = {
+  method: 'localization.additionalInfo',
+  validationJsonPath: 'result',
+  content: { '': '', defaultKey: 'defaultValue=' },
+};
+
+exports.EMPTY_KEY_VALUE_FOR_LOCALIZATION_ADDITIONALINFO = {
+  method: 'localization.additionalInfo',
+  validationJsonPath: 'result',
+  content: { '': '' },
+};
+
+exports.EMPTY_RESPONSE_FOR_LOCALIZATION_ADDITIONALINFO = {
+  method: 'localization.additionalInfo',
+  validationJsonPath: 'result',
+  content: {},
+};
+
+exports.GET_LOCALIZATION_LATLON = {
+  method: 'localization.latlon',
+  params: {},
+};
+
+exports.EXPECTED_LOCALIZATION_LATLON = {
+  method: 'localization.latlon',
+  validationJsonPath: 'result',
+  content: [39.9549, 75.1699],
+};
```

Comment:
can we say that latlon value be same as in line 171 ?

Reply:
In Xumo, latlon is an exceptional method. Updated its content to an empty string. For other config modules, we can override the validationObject if needed.